### ISSUE TITLE
How did I build openresty at nginx packages

### ISSUE THREAD

**Issue by bboyjunzi** [openwrt/packages#20232]
How did I build luajit within nginx

**Comment by Spudz76** [comment]
openresty is a full fork of nginx and is not packaged for OpenWRT.  The luajit extension is not available as just-a-plugin to add on to the normal nginx source.

someone (you?) must make a package for openresty if you want it.  probably easy since most of what it would need is already in the nginx package (mainly swap out the sources and maybe add some extra stuff to the configure phase).

**Comment by Spudz76** [comment]
I did find this in the menuconfig, maybe it does what you need?
![image](https://user-images.githubusercontent.com/2391234/211279838-b79ca679-1520-4012-b40e-6f6eaf98226c.png)

it will not be in the prebuilds so to try it you must compile your own

pretty sure it's not quite the same as luajit though

I haven't bothered with openresty in years.  The fact they don't just make it a plugin is stupid (and they claim they aren't a fork when what the heck else would it be called?)

**Comment by bboyjunzi** [comment]
I'm working at 
https://github.com/openresty/lua-resty-websocket/tree/master
beacuase the openresty need to run luajit + nginx . but nginx of openwrt package was lua+nginx.
So I raise a question, nobody go forward at openwrt nginx.

**Comment by Spudz76** [comment]
yes there must be an openresty package created, because it's a ["not-a-fork" full fork of nginx](https://openresty.org/)

**Comment by brada4** [comment]
> I'm working at https://github.com/openresty/lua-resty-websocket/tree/master beacuase the openresty need to run luajit + nginx . but nginx of openwrt package was lua+nginx. So I raise a question, nobody go forward at openwrt nginx.

Seems real stuff, just that uses no-jit system default lua interpreter in place of openresty forked luajit.
https://github.com/openwrt/packages/blob/88b0e30806965a73058d79dd2d8bcbe6a2da88f9/net/nginx/Makefile#L498
